### DISCUSSION TITLE
RobotPower With Time Delay Needed

### DISCUSSION THREAD

**Discussion by slowrunner** [iRobotEducation/create3_docs#254] [Ideas]
When I want to shut down my robot, issuing a call to the /RobotPower service will do the trick for shutting everything down, BUT it jerks the power out from under the robot's RaspberryPi4.  Very often this corrupts my Python logging life.log.

If RobotPower service took a delay: 10 (seconds) parameter, it would give me time to kill my ROS nodes and issue a ```sudo shutdown -h now``` on the Pi4.

**Comment by shamlian** [comment]
Created an issue (#280) for this request.

**Comment by slowrunner** [comment]
I no longer have a TB4lite, and intend to run my new Create3-WaLi robot 24/7 so shutdown is no longer an issue for me.

Closing my side of this issue - I see @shamlian opened an issue for this so it will not be lost by closing this thread.
